### ISSUE TITLE
Poll: What package format should gqrx be made available in next?

### ISSUE THREAD

**Issue by ei8fdb** [gqrx-sdr/gqrx#1122]
(This is not a support question, it's more to get feedback from the gqrx community. I'll leave the poll open for a few months. And I'll share [the results on my blog](https://www.ei8fdb.org/thoughts/2022/04/27/what-package-format-should-gqrx-be-made-available-in-next/).)

Right now gqrx is available as an AppImage, as a Mac OS dmg, and as sourcecode. It’d be nice to have it available in other package formats.

You can vote on what package format to address next by filling out this quick 1 question poll:

[https://poll.fm/11102781](https://poll.fm/11102781)

**Comment by ryanvolz** [comment]
I would like to add that it is also already available as a [conda package for a few different platforms](https://anaconda.org/conda-forge/gqrx): linux-64, linux-aarch64, osx-64, and win-64. As soon as conda-forge gets its act together and gets full support for Qt for osx-arm64 and linux-ppc64le, those platforms will also be added. The function of these overlaps with the poll options, so I thought it was worth mentioning, but I don't mean to discourage packaging gqrx in more formats!

**Comment by argilo** [comment]
Likewise, Gqrx is also available in the package manager of various Linux distributions, although often an older version: https://repology.org/project/gqrx/packages

Conda is the option I recommend to Windows users today, but it's quite heavy if you only want to install one thing.

**Comment by ei8fdb** [comment]
@argilo @ryanvolz would it help if I modified the options available in the poll? I'm happy to do that. I've cast my personal vote, but the majority vote should take preference.

**Comment by argilo** [comment]
@ei8fdb Those options already exist today, so they don't need to be in the poll.

I see one respondent already suggested a .deb package (which could be used on Debian-based distributions). It might be worth adding that one to the list.

**Comment by ei8fdb** [comment]
> @ei8fdb Those options already exist today, so they don't need to be in the poll.
 
👍

> I see one respondent already suggested a .deb package (which could be used on Debian-based distributions). It might be worth adding that one to the list.

Ok I'll try and add a .deb option.

**Comment by ryanvolz** [comment]
Yeah, I mostly wanted to head off respondents suggesting "other" things that already existed! The current poll options are quite valuable for getting feedback on where to make the installation process easier.

**Comment by ei8fdb** [comment]
> Yeah, I mostly wanted to head off respondents suggesting "other" things that already existed! The current poll options are quite valuable for getting feedback on where to make the installation process easier.

Would it help to update documentation with the other installation options? I'd be happy to do that.

**Comment by argilo** [comment]
Yes, I think that would be helpful. The readme is very light on details at the moment: https://github.com/gqrx-sdr/gqrx#download

The web site has a few details specific to the Raspberry Pi: https://gqrx.dk/download/gqrx-sdr-for-the-raspberry-pi

**Comment by barjac** [comment]
I maintain gqrx in the Mageia Linux distribution and have done for many years. Surely other distributions do the same. Using the distribution package is by far the simplest and most secure and reliable way to install any software.

**Comment by argilo** [comment]
> Using the distribution package is by far the simplest and most secure and reliable way to install any software.

Absolutely. But one drawback is that the distribution package may be old—up to two years in the case of Ubuntu LTS or Debian. Alternate formats (such as the existing AppImage releases) are convenient for users wishing to run the latest release.

**Comment by ei8fdb** [comment]
> I maintain gqrx in the Mageia Linux distribution and have done for many years

@barjac could you share the links/location of the installation docs for gqrx on Mageia Linux and I'll add it to the installation docs. Or alternatively, you can submit a PR to update the docs.

**Comment by barjac** [comment]
In Mageia 8 which is the current stable release (https://www.mageia.org/en/8/) gqrx is available in the package manager from the core repository which is enabled by default.
The easily accessed Mageia Control Centre (aka MCC) is where the package manager resides.
Or from the command line it can be installed (as root) using:
urpmi gqrx
That's it, nothing more needed!

The versions of packages available in Mageia are the latest stable at the time of the distribution release, unless there are security or bug fixes that require a version update. If the very latest versions are required then we do maintain a backports repository and requests for backports will be considered by package maintainers where practicable.

I hope that helps
